addons/website/data/digest_data.xml

```diff
@@ -4,7 +4,7 @@
         <record id="digest_tip_website_0" model="digest.tip">
             <field name="name">Tip: Engage with visitors to convert them into leads</field>
             <field name="sequence">1500</field>
-            <field name="group_id" ref="website.group_website_publisher" />
+            <field name="group_id" ref="website.group_website_restricted_editor" />
```

Comment:
Since you had to touch those lines, I would have removed the space before "/>" at the same time :smile: But I guess it's a matter of taste :)